demos/logging-stack/logging_stack.h

```diff
@@ -41,7 +41,9 @@
 
 /* Common macro for all logging interface macros. */
 /* TODO - Replace printf with an implementation function. */
-#define SdkLog( string )    printf string
+#ifndef SdkLog
+    #define SdkLog( string )    printf string
+#endif
```

Comment:
This was changed so that we can easily disable logging from the command line with `-D'SdkLog(string)='`

Reply:
This is nice, but I think it may be better to have a "DISABLE_LOGGING" master macro or something?

Reply:
Changed